src/sidebar.jsx

```diff
@@ -334,6 +338,17 @@ const DropdownWithKebab = ({
             },
             title: _("Rename")
         },
+        ...(selected.length === 1 && selected[0].type === "reg")
+
+            ? [
+                { type: "divider" },
+                {
+                    id: "download-item",
+                    onClick: () => downloadFile(currentPath, selected[0]),
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-175-20240131-121935-079ef859-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)